### PR TITLE
Read all csvs

### PR DESCRIPTION
**Pull request by acnb**
In my last pull request I simplified the csv-file exported from QuantaSoft (v1.7.4). After the export, we used to ad a ',dummy' to the first row. 

This pull request now includes code to do this automatically. Files from other QuantaSoft versions should work as well.

### CODE REVIEW

**Comment by acnb** [comment]
Thanks for the comments. I hope this commit has clearer comments. 
Andreas

**Comment by acnb** [comment]
sorry for accidentally closing...

File: R/read.R [hunk 1]
```diff
@@ -89,24 +89,30 @@ read_files <- function(plate, data_files, meta_file) {
   # Read the metadata file if one was given
   if (!is.null(meta_file)) {
     tryCatch({
+      # read meta header
       cn <- 
         utils::read.csv(meta_file, stringsAsFactors = FALSE, 
-                        nrows=1, header = F) %>%
-        unlist(.[1,]) %>%
+                        nrows = 1, header = FALSE) %>%
+        head() %>%
```
Comment (daattali) [inline R/read.R]: should be head(1), no?

Comment (acnb) [inline R/read.R]: It works because there is only one row anyway. Maybe we should remove head()?

**Comment by daattali** [comment]
If head is unnecessary then yes remove it :) Should be as simple as
possible while still correct!

---
Founder & Lead Consultant
AttaliTech Ltd
http://attalitech.com

**Comment by daattali** [comment]
Thanks!

**Pull request merged**